src/components/Home/index.tsx

```diff
@@ -66,6 +91,12 @@ export default function HomePage(): ReactElement {
       />
 
       <SectionQueryResult title="Most Sales" query={queryMostSales} />
+      <SectionQueryResult
+        title="Most Viewed"
+        query={queryMostViewed}
+        queryData={mostViewedDids}
+        tooltip="Assets from all supported chains. It is not influenced by selected networks"
```

Comment:
Not sure why we're not filtering by selected networks. I think that's a valuable information to have 👍🏽

Reply:
Can't . This information is from netlify and it's limited. For most chains i would have 0 results.

Reply:
Or maybe i can do something

Reply:
nope way too complicated